### ISSUE TITLE
lower the extraordinary and legendary drop chances

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#209]
```
Perhaps we should tweak the extraordinary and legendary drop chances?
viewtopic.php?f=2&t=703

Many members did state the current low drop rate is frustrating.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 9 Jul 2011 at 6:07

**Comment by GoogleCodeExporter** [comment]
```
Should be closed as the skill Magic finder is there.
```

Original comment by `SamuelPl...@gmail.com` on 21 Jul 2011 at 8:46

**Comment by GoogleCodeExporter** [comment]
```
Just realized on the drop chance for legendary items when it shows 1/10,000 
chance that really means that really changes into 1/1,000,000. For example its 
(5)÷100 chance to get a rock. While the rapier lifesteal is(1/10000)÷100.
{rock|1|1|5|}
{rapier_lifesteal|1|1|1/10000|}
```

Original comment by `dope...@gmail.com` on 2 Oct 2011 at 7:26

**Comment by GoogleCodeExporter** [comment]
```
I think it would be a good idea if you incressed the chances of getting a 
legendary items. I've gone up 5 levels trying to get the Rapier of Lifesteal 
and i'm level 41 now. I understand making things rare but making them that rare 
is more like trying to find a needle in a haystack. I think it would be a 
better idea if you made it (1/100)/100.
```

Original comment by `dope...@gmail.com` on 6 Oct 2011 at 5:06

**Comment by GoogleCodeExporter** [comment]
```
> 1/10,000 chance that really means that really changes into 1/1,000,000. For 
example its (5)÷100 chance to get a rock. While the rapier lifesteal 
is(1/10000)÷100.

Thats wrong. There is different code for "1/10000". Its like: "1:10000"

Have a look here:
http://andors.techby2guys.com/viewtopic.php?f=2&t=703
```

Original comment by `SamuelPl...@gmail.com` on 6 Oct 2011 at 5:32

**Comment by GoogleCodeExporter** [comment]
```
didn't know that thank you very much
```

Original comment by `dope...@gmail.com` on 6 Oct 2011 at 6:37

**Comment by GoogleCodeExporter** [comment]
```
@Oskar:
Should be closed as the skill Magic finder is there in 0.6.10.
```

Original comment by `SamuelPl...@gmail.com` on 6 Oct 2011 at 6:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `sdeva...@gmail.com` on 6 Oct 2011 at 6:49
- Changed state: **Fixed**
- Added labels: **Milestone-0.6.10**
